.scalafix.conf

```diff
@@ -0,0 +1,16 @@
+rules = [
+  RemoveUnused
+  LeakingImplicitClassVal
+  ProcedureSyntax
+  NoValInForComprehension
+  SortImports
+]
+
+SortImports.blocks = [
+  "*",
+  "io.",
+  "org.",
+  "java.",
+  "scala.",
+  "zio."
+]
```

Comment:
Perhaps we can drop the io / org separation

Reply:
From my comment on the original issue:

>  I prefer the order "java, scala, libraries, unknown, local" in my projects.
> So here we would end up with
> 
> SortImports.blocks = [
>   "java.",
>   "scala.",
>   "*",
>   "zio."
> ]
> 
> But I think that is more personal preference than anything :)

Reply:
@NeQuissimus Yeah I will adapt to your suggestion. TBH I am not too strongly opinionated on this